code/linear_svm.py

```diff
@@ -0,0 +1,54 @@
+#All the libraries we need for linear SVM
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn import svm
+#This is used for our dataset
+from sklearn.datasets import load_breast_cancer
+
+
+# =============================================================================
+# We are using sklearn datasets to create the set of data points about breast cancer
+# Data is the set data points
+# target is the classification of those data points. 
+# More information can be found athttps://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_breast_cancer.html#sklearn.datasets.load_breast_cancer
+# =============================================================================
+dataCancer = load_breast_cancer()
+data = dataCancer.data[:, :2]
```

Comment:
What does the `[:, :2]` do? Probably a good idea to explain that or rewrite it in a clearer way